README.md

```diff
@@ -16,22 +59,56 @@ The [v4_beta_deprecated](https://github.com/aws/aws-iot-device-sdk-embedded-C/tr
 
 All of the released versions of the AWS IoT Embedded C SDK libraries are available as git tags. For example, the last release of the v3 SDK version is available at [tag 3.0.1](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.0.1).
 
+## Versioning
+
+C-SDK releases will now follow a date based versioning scheme with the format YYYYMM.NN_major, where: 
+
+* Y represents the year.
+* M represents the month.
+* N represents the release order within the designated month (00 being the first release).
+* A "major" denotation indicates the addition of new features or significant updates to multiple libraries. 
+
+For example, a second release in June 2021 would be 202106.01. Although the SDK releases have moved to date-based versioning, each library within the SDK will still retain semantic versioning. In semantic versioning, the version number itself (X.Y.Z) indicates whether the release is a major, minor, or point release. You can use the semantic version of a library to assess the scope and impact of a new release on your application. 
+
+## Cloning
+This repo uses [Git Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) to bring in dependent components.
+
+Note: If you download the ZIP file provided by GitHub UI, you will not get the contents of the submodules. (The ZIP file is also not a valid git repository)  If you download from the [Tags](https://github.com/aws/aws-iot-device-sdk-embedded-C/tags) or [Releases](https://github.com/aws/aws-iot-device-sdk-embedded-C/releases), you will get the entire repository (including the submodules)
```

Comment:
I think the period goes after the parentheses. Also, should we specify which zip includes the entire repository? When there's a GitHub release, GitHub also has a "Source code (zip)" in addition to whatever files we upload as assets, and I think that one is identical to the one you get from the GitHub UI (e.g. [FreeRTOS](https://github.com/FreeRTOS/FreeRTOS/releases))
```suggestion
Note: If you download the ZIP file provided by GitHub UI, you will not get the contents of the submodules (The ZIP file is also not a valid git repository).  If you download from the [Tags](https://github.com/aws/aws-iot-device-sdk-embedded-C/tags) or [Releases](https://github.com/aws/aws-iot-device-sdk-embedded-C/releases), you will get the entire repository (including the submodules)
```

Reply:
I agree that we should clearly mention that the "Source code (zip)" and "Source code (tar.gz)" assets that are auto-uploaded by GitHub do not contain the submodule code. 
We should also mention the name of the assert that we plan to upload, which will be named "aws-iot-device-sdk-embedded-c-202009.00.zip".
@leegeth, can you confirm the name of the new asset that we will upload?

Reply:
Thanks Archit. That is a good point. The asset name is **aws-iot-device-sdk-embedded-C-202009.00.zip**